.github/dependabot.yml

```diff
@@ -3,7 +3,8 @@ updates:
   - package-ecosystem: cargo
     directory: "/"
     schedule:
-      interval: "weekly"
+      interval: "daily"
```

Comment:
I think weekly is too much.
Why do we need daily?

Reply:
I thought having this action to auto-merge dependabot PRs would grealty reduce the churn and thus shouldn't put more burden on maintainers even if we change the frequency to daily.

Reply:
I am happy to revert it back to weekly

Reply:
If you do it daily, you will have the added burden of writing release notes. Consideration should be given to that.
Youki prevents this by generating release notes with labels.